Web/Html,CSS,Javascript/github-profiles/script.js

```diff
@@ -36,7 +37,7 @@ function createUserCard(user) {
       <img src="${user.avatar_url}" alt="${user.name}" class="avatar">
     </div>
     <div class="user-info">
-      <h2>${userID}</h2>
+      <a style="color:rgb(238,238,238); text-decoration:none;" href="${userURL}"><h2>${userID}</h2></a>
```

Comment:
You just have to redirect to the link: www.github.com/<username> where username can be extracted from user object.

Reply:
<img width="1440" alt="Screenshot 2022-10-04 at 11 00 58 AM" src="https://user-images.githubusercontent.com/92316166/193742221-f15697b5-b7b8-4a61-89ce-8b3bedf994f5.png">


Yes, first I thought would have somewhat do something like adding username in "www.github.com/" by extracting it from user object but when I checked API it was providing direct link for that and found that with "html_url" in the user object.
It's working properly I have tested it.